NEWS.md

```diff
@@ -1,10 +1,14 @@
 # NEWS
 
-## polars (development version)
+## Polars R Package (development version)
+
+## Polars R Package 0.16.2
 
 ### New features
 
 - `$cut()` and `$qcut()` to bin continuous values into discrete categories (#1057).
+- `pl$scan_parquet()` and `pl$read_parquet()` can read data from the internet by specifying a URL
+  to the first argument (#1056, @andyquinterom).
```

Comment:
That was already the case, no?

Reply:
No, there used to be panic when trying to read Parquet from a URL.
Perhaps this feature was necessary.
(I've been so busy the last few weeks I haven't had time to point this out)

https://github.com/pola-rs/r-polars/blob/4a81740721ae7a2d5270656b081d3ee11a6ad6b4/src/rust/Cargo.toml#L104